apps/src/code-studio/initApp/project.js

```diff
@@ -542,21 +542,14 @@ var projects = module.exports = {
     if (typeof arguments[0] === 'function' || !sourceAndHtml) {
       // If no source is provided, shift the arguments and ask for the source
       // ourselves.
-      var args = Array.prototype.slice.apply(arguments);
+      const args = Array.prototype.slice.apply(arguments);
```

Comment:
Not your change, but in ES6 land, I believe use of `arguments` is generally discouraged, and one is instead encouraged to use destructuring, i.e. `function foo(...args)`. This case is a little stranger in that we both want friendly names, and to access via args.

I wonder if the right way to do this would be to have something like:
```
save(...args) {
  let [sourceAndHtml, callBack, forceNewVersion, preparingRemix, ...otherArgs] = args;
}
```
Not sure that's a lot better. In any case, feel free to ignore this as you wish as this is really tangential to the changes you made here.